tests/test_ext_autodoc.py

```diff
@@ -1563,8 +1563,8 @@ def test_singledispatch(app):
         '',
         '',
         '.. py:function:: func(arg, kwarg=None)',
-        '   func(arg: int, kwarg=None)',
-        '   func(arg: str, kwarg=None)',
+        '                 func(arg: int, kwarg=None)',
+        '                 func(arg: str, kwarg=None)',
         '   :module: target.singledispatch',
```

Comment:
I don't know enough about rST to know if this whitespace is fine / a good idea. Happy to revert this bit.

Reply:
I did not know reST support such indentation. But it works fine on my local. I prefer this. +1!